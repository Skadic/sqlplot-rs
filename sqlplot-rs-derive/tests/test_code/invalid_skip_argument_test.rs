use sqlplot_rs_derive::ResultLine;

#[allow(unused)]
#[derive(ResultLine)]
struct MyStruct {
    a: &'static str,
    #[skip(random_arg)]
    b: u32,
    c: f64,
}

fn main() {}