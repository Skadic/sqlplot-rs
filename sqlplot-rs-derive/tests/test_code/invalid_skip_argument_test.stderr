error: unexpected token in attribute
 --> tests/test_code/invalid_skip_argument_test.rs:7:11
  |
7 |     #[skip(random_arg)]
  |           ^
